streampark-console/streampark-console-webapp/src/store/modules/user.js

```diff
@@ -61,6 +61,10 @@ const user = {
       storage.set(USER_ROUTER, routers)
       state.routers = routers
```

Comment:
Should we use sessionStorage here? Mutil page shouldn't share the same router, right?

Reply:
> Should we use sessionStorage here? Mutil page shouldn't share the same router, right?

oh, yes. you are right.